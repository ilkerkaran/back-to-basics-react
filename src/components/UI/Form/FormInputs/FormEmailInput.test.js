import React from 'react';
import renderer from 'react-test-renderer';
import Email from './FormEmailInput';


describe('<Email /> snapshot ', () => {
  const inputConfig = {
    inputType: 'email',
    inputName: 'email',
    label: 'Username',
    isRequired: true,
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<Email inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
