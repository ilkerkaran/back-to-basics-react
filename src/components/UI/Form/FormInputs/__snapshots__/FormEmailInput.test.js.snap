// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Email /> snapshot  renders correctly 1`] = `
<div>
  <input
    className="Input"
    name="email"
    placeholder="Username"
    type="text"
  />
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
